src/chainstate/stacks/index/node.rs

```diff
@@ -1425,24 +1435,29 @@ mod test {
         for i in 0..3 {
             assert!(node4.insert(&TriePtr::new(TrieNodeID::Node16, (i+1) as u8, (i+2) as u32)));
         }
-        let node4_bytes = vec![
+        let mut node4_bytes = vec![
             // node ID
             TrieNodeID::Node4,
-            // ptrs (4)
-            TrieNodeID::Node16, 0x01, 0, 0, 0, 0, 
-            TrieNodeID::Node16, 0x02, 0, 0, 0, 0, 
-            TrieNodeID::Node16, 0x03, 0, 0, 0, 0, 
-            TrieNodeID::Empty, 0x00, 0, 0, 0, 0, 
+        ];
```

Comment:
Please revert this.  It's much easier for a Jude-brain to comprehend a raw array of numbers than a sequence of `extend_from_slice` calls.  It also makes comparing outputs easier if there's a serialization bug.

Reply:
Addressed in b8594a21